src/main/java/uk/gov/hmcts/reform/roleassignment/domain/model/RoleConfig.java

```diff
@@ -38,6 +38,7 @@ private static void setRoleNameAndCategory(RoleConfigRole role) {
             p -> {
                 p.setRoleName(roleName);
                 p.setRoleCategory(roleCategory);
+                p.setSubstantive(role.isSubstantive());
```

Comment:
We need to updated the method name and the comments above to signify we are replicating all the common fields on pattern. Name should be "setCommonFields"

Reply:
Renamed the method